src/Controls/src/Core/IWindowCreator.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Microsoft.Maui.Controls;
+
+public interface IWindowCreator
+{
+	Window CreateWindow(Application app, IActivationState? activationState);
```

Comment:
Are we planning more options for .NET 8?. Like have the Windows collection etc. Ig that is the case, would prefer a better naming like `IWindowManager`.

Reply:
The root of the original issue was being able to handle the CreateWindow. This PR ultimately closes that issue and nullifies the requirement from my perspective as a 3rd party maintainer.